css/css-backgrounds/border-radius-clipping.html

```diff
@@ -1,40 +1,38 @@
 <!DOCTYPE html>
-<html>
-<head>
-    <title>Border radius clipping</title>
-    <link rel="author" title="zhouli" href="mailto:liz@oupeng.com">
-    <link rel="help" href="http://www.w3.org/TR/css3-background/#corner-clipping" />
-    <link rel="match" href="reference/border-radius-clipping-ref.html">
-    <meta name="assert" content="When set with 'overflow: hidden', outer container should clip background of an element insides at the corners. " />
-    <style>
-        #parent{
-            border-radius: 15px;
-            border: 1px solid blue;
-            overflow: hidden;
-            width: 300px;
-            height: 30px;
-        }
-        #son{
-            background-color: green;
-            width: 300px;
-            height: 30px;
-        }
-        #dummy {
-            border-radius: 5000em;
-            height: 10000em;
-            overflow: hidden;
-            width: 10000em;
-        }
-    </style>
-</head>
-<body>
-<div id="parent">
+
+  <meta charset="UTF-8">
+
+  <title>CSS Backgrounds and Borders Test: 'border-radius' and corner-clipping of content elements</title>
+
+  <link rel="author" title="zhouli" href="mailto:liz@oupeng.com">
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="http://www.w3.org/TR/css3-background/#corner-clipping">
+  <link rel="match" href="reference/border-radius-clipping-ref.html">
+
+  <meta content="" name="flag">
+  <meta name="assert" content="When set with 'overflow: hidden', an outer container must clip background of its child element at its corners. In this test, the outer container has a 'border-radius' declaration which implies that the clipping must occur at its curves.">
+
+  <style>
+  div#parent
+    {
+      background-color: red;
+      border-radius: 50%;
+      height: 100px;
+      overflow: hidden;
+      width: 100px;
```

Comment:
This would be a stronger test if we include a transparent border and padding, to make sure that it is clipping at the correct edge.

Reply:
I spent considerable time trying to achieve your requested improvements and here is my best effort:

http://www.gtalbot.org/BrowserBugsSection/CSS3Backgrounds/border-radius-clipping-GT.html

(will be) Reference:
http://www.gtalbot.org/BrowserBugsSection/CSS3Backgrounds/reference/border-radius-clipping-ref.html

A pull request coming based on these 2 files coming up...